client/js/views/templates/AboutUs.js

```diff
@@ -1,3 +1,24 @@
 module.exports = function( { model } ) {
-    return `<div>about us</div>`
+	const images = model.reduce((memo, image) => {
+	  	const markup = `<div class='${ image.class }'>
+	    	<img data-src='${ this.ImageSrc(image.filename) }' alt='${ image.alt }'/> 
+	    	<div>${ image.date }<br/>${ image.caption }</div>
```

Comment:
Instead of `<br>`, just have two nested divs, one for the date and one for the caption.

Reply:
I'm okay with <br/> as long as the CSS doesn't get complicated.  Curious to why you prefer to use two `<div>`s @ScottAP108 ?

Reply:
Strictly speaking, html should be used semantically, not for presentation purposes (that's css's job). So a `<br>` makes sense for an address or a poem or something like that. I guess you could make an argument for this case, though. Not a huge deal really, but I definitely wouldn't use them to force line breaks in a paragraph, unless that paragraph is a poem.